internal/core/src/plan/PlanNode.h

```diff
@@ -243,6 +248,13 @@ class FilterBitsNode : public PlanNode {
                            filter_->ToString());
     }
 
+    expr::ExprInfo
+    GatherInfo() const override {
```

Comment:
void GatherInfo(expr::ExprInfo&) is better, avoid copy

Reply:
This does not copy because of RVO